scripts/jenkins-acceptance-tests.sh

```diff
@@ -37,3 +37,4 @@ pip install -r requirements/test-acceptance.txt
 
 cd test/acceptance
 python tests.py
+SELENIUM_BROWSER=phantomjs python accessibility.py
```

Comment:
Should probably give a heads up to Usman, who is currently on the hook for any failures in this pipeline.